src/main/webapp/view/view.js

```diff
@@ -12,66 +12,75 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import Synchronizer from '../synchronizer.js';
-
-import {intializeDiscussion} from '../view/discussion/discussion.js';
+import DiscussionArea from './discussion/discussion-area.js';
 import TranscriptArea from './transcript/transcript-area.js';
 import Video from './video/video.js';
 
 const ENDPOINT_LECTURE = '/lecture';
 
 const PARAM_ID = 'id';
 
-// TODO: Remove global scope and add to view object.
-window.video = new Video();
+export default class View {
```

Comment:
can we rename this `LectureView`? I think `View` is too generic

Reply:
renamed